clin-workflows/es-index-templates/apply_base_template.sh

```diff
@@ -0,0 +1,14 @@
+if [ -z "$ES_URL" ]; then
+    echo "ES_URL environment variable is required, but not defined.";
+    exit 1;
+fi
+
+curl -f -XPUT -H "Content-Type: application/json" "$ES_URL/_template/base_template" -d '
+{
+  "index_patterns": ["*"],
+  "settings": {
+    "number_of_shards": 3,
```

Comment:
I don't know about this...  Each indices have their own settings right now ?!

Reply:
See our latest discussion on Slack. I'll address it in a separate PR and keep the current strategy for shards right now.